cockpit/src/subscriptions-register.jsx

```diff
@@ -126,9 +111,9 @@ var PatternDialogBody = React.createClass({
                         </td>
                         <td>
                             <Select key='urlSource' onChange={ this.props.onChange.bind(this, 'url') }
-                                    id="subscription-register-url" initial="default">
-                                <SelectEntry data='default' key='default'>{ urlEntries['default'] }</SelectEntry>
-                                <SelectEntry data='custom' key='custom'>{ urlEntries['custom'] }</SelectEntry>
+                                    id="subscription-register-url">
+                                <option value="default">{ urlEntries['default'] }</option>
+                                <option value="custom">{ urlEntries['custom'] }</option>
```

Comment:
I can see some errors in Javascript console, because these options aren't React entities. Are you going to change it, when this will take real effect? In this case please add some `TODO`/`FIXME` comment.

Reply:
I did not see these, but I did see that the error message was erring when it wasn't a String, so I changed that (see changes to subscriptions-view.jsx). Maybe that's what you were seeing?

Reply:
I cannot see this error anymore. It probably fixed this problem.